.github/workflows/ci.yml

```
@@ -25,17 +25,36 @@ jobs:
         - linux: py310
         - linux: py311-cov
           coverage: codecov
-        - linux: py312
+        - linux: py312-withromancal-cov
+          coverage: codecov
         - linux: py313
         - macos: py311
   test_upstream:
     uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@924441154cf3053034c6513d5e06c69d262fb9a6  # v1.13.0
     with:
       envs: |
         - linux: rad
-  test_with_romancal:
+
+  environment:
+    runs-on: ubuntu-latest
+    steps:
+      - id: data_path
+        run: echo "path=${{ runner.temp }}/data" >> $GITHUB_OUTPUT
+    outputs:
+      data_path: ${{ steps.data_path.outputs.path }}
+  crds_contexts:
+    uses: ./.github/workflows/contexts.yml
```

we should use the one at CRDS instead (to outsource the maintenance there)
```suggestion
    uses: spacetelescope/crds/.github/workflows/contexts.yml@master
```